packages/webhooks/tasks/sendWebhook.js

```diff
@@ -126,6 +126,7 @@ async function sendWebhook (subscriptionId) {
             }
         }
 
+        // NOTE: we don't need to update WebhookSubscription if no any updates found!
```

Comment:
need to fix this logic!

Reply:
we also need to reset the counter

Reply:
Please read test before touching my piece of code! 
https://github.com/open-condo-software/condo/blob/master/packages/webhooks/schema/models/WebhookSubscription.js#L87

https://github.com/open-condo-software/condo/blob/master/packages/webhooks/schema/models/WebhookSubscription.js#L110